lib/simp/cli/config/items/data/svckill_mode.rb

```diff
@@ -0,0 +1,64 @@
+require File.expand_path( '../item', File.dirname(__FILE__) )
+
+module Simp; end
+class Simp::Cli; end
+module Simp::Cli::Config
+  class Item::SvckillMode < Item
+    def initialize
+      super
+      @key         = 'svckill::mode'
+      @description = %Q{
+Strategy svckill should use when it encounters undeclared services
+
+'enforcing' = Shut down and disable all services not listed in your
+              manifests or the exclusion file
+'warning'   = Only report what undeclared services should be shut
+              down and disabled, without actually making the changes
+              to the system
+
+NOTICE: svckill is the mechanism that SIMP uses to comply with the
+requirement that no unauthorized services are running on your system.
+Is it HIGHLY recommended that you set this to 'enforcing'. Please be
+aware that, if you do this, svckill will stop ALL services that are
+not referenced in your Puppet configuration.}
+
+      @warning_msgs = {
```

Comment:
We might consider having users 'press any key to continue' so this does not fly by on the screen after application